examples/vite-react/src/components/BreadcrumbDemo.tsx

```diff
@@ -0,0 +1,22 @@
+import { Breadcrumb, Icon } from "@animareflection/ui/client";
+import { FiChevronRight as ChevronRightIcon } from "react-icons/fi";
+
+import { Wrapper } from "components";
+
+const BreadcrumbDemo = () => {
+  // Note: To obtain the pathname in vite-react, use `useLocation()` from "react-router-dom".
+  // Ensure the App is wrapped in `BrowserRouter`.
```

Comment:
This can be removed, since this is specific to React Router, which people don't have to use for Vite React apps